integration/README.md

```diff
@@ -0,0 +1,66 @@
+
+After a successful deployment, each google-cloud-ruby libraries can then run its integration tests again the deployed sample application. For example, the [google-cloud-logging tests](google-cloud-logging/integration/) send HTTP requests to redefined routes on the sample application to trigger logs being generated. Then it's able to verify the result logs through gcloud sdk. 
+
+### Integration tests on Google Container Engine
+To run the integration tests on Google Container Engine:
+```sh
+$ rake integration:gke
+```
+Similar to the workflow of `integration:gae` task, the `integration:gke` task uses the Dockerfile.example file from each sample application as the template to build a docker image, deploy it to Google Container Registry, and deploy Google Container pods to launch the application. In the case of GKE pods configuration, all sample application shares a same configuration defined in [integration_rc.yaml.example](integration_rc.yaml.example) file.
+
+After the integration tests finish run against the deployed GKE sample application, the `integration:gke` application is also able to clean up all the resources created during the deployment. These resources include local test docker image, GCR test docker image, GKE ReplicationController, and GKE pods.  
+
+### Integration tests on Google Compute Engine
+We're currently not supporting automated tests on Google Compute Engine environment for the following reasons:
+* Major functionalities such as metadata service accisibility, application default authentication are, operating system, and API accessibility are already covered by other automated tests.
+* Both GAE and GKE are currently hosted on GCE VMs.
+* It's harder to automate deployment on GCE environment than GAE or GKE, due to GCE's IaaS nature.
+
+### Write tests
+
+Each google-cloud-ruby libraries should define its own set of integration tests. It it doesn't already, make sure a corresponding `integration:gae` or `integration:gke` task exists at the package level. These tasks will be invoked by top level `integration` task when deployment is finished. Otherwise these package level tasks can also be invoked independently assume deployment already happened.
+
+Common test cases defined in google-cloud-*/integration/ directory will be run on both GAE and GKE environments. Environment specific test cases can be defined in google-cloud-*/integration/gae/ or google-cloud-*/integration/gke/ directories. Such test cases will only be run against the corresponding environment.
```

Comment:
There are a number of file names in this document that are missing backticks and should be fixed. The file paths in this line must be wrapped in backticks in order to avoid the `*` characters being interpreted as markdown.